src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: Function;
+}
 
+function squareFloor(square: number): number {
+  return Math.floor(square * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape;
```

Comment:
set the corresponding value here, fix it everywhere
```suggestion
  shape = Shape.Triangle;
```